Enhance documentation on accessing hidden SO types (#199046)
elastic/kibana: dev_docs/tutorials/saved_objects.mdx

that objects of this type can only exist in a single space. See
<DocLink id="kibDevDocsSavedObjectsIntro" section="sharing-saved-objects" text="Sharing Saved Objects"/> for more information.

> [3] This field determines whether repositories have access to the type by default. he hidden types will not be automatically exposed via the HTTP API.

Reviewer: ```suggestion
[3] This field determines whether repositories have access to the type by default. Hidden types will not be automatically exposed via the Saved Objects Client APIs.
```

> Access a hidden type with a client that explicitly includes a list of hidden types `SavedObjectsClientProviderOptions[includedHiddenTypes]`.

Reviewer: ```suggestion
Hidden types must be listed in `SavedObjectsClientProviderOptions[includedHiddenTypes]` to be accessible by the client.
```
